modules/apps/frontend-data-set/frontend-data-set-views-web/types/src/main/resources/META-INF/resources/js/components/OrderableTable.d.ts

```diff
@@ -12,21 +12,26 @@
  * details.
  */
 
-/// <reference types="react" />
```

Comment:
i still don't know why these `///` references get added in and then removed... Maybe this is fine though? I'd just confirm with an `ant all` to make sure it works without error.

Reply:
Ah yes, I've been noticing this as well, but I try not to commit the removals. I'll go ahead and remove this change (keeping `/// <reference types="react" />` there since the other `*.d.ts` files have that as well).

Reply:
I ended up leaving this change it `yarn types` makes this change. I'm assuming it's not needed because `import React from 'react';` is added for the `React.FC` type.

Reply:
Also confirmed that `ant all` still works with no errors.